x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_alert_group_from_sequence.test.ts

```diff
@@ -146,6 +142,64 @@ describe('buildAlert', () => {
   });
 
   describe('recursive intersection between objects', () => {
+    describe('objectPairIntersection', () => {
+      test('should work with arrays with exact matches', () => {
+        const a = {
+          field1: [1],
+        };
+        const b = {
+          field1: [1],
+        };
+        const intersection = objectPairIntersection(a, b);
+        const expected = {
+          field1: [1],
+        };
+        expect(intersection).toEqual(expected);
+      });
+
+      test('should work with arrays with differing lengths', () => {
```

Comment:
Again, "work" is such an ambiguous verb as to not be useful here:

```suggestion
      test('returns the intersection of arrays with differing lengths', () => {
```